.github/workflows/windows-backup-test.yml

```
@@ -0,0 +1,119 @@
+      - 'packages/cli/tests/config-engine.test.ts'
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'packages/cli/src/core/config-engine.ts'
+      - 'packages/cli/tests/config-engine.test.ts'
+  workflow_dispatch:
+
+jobs:
+  windows-backup-test:
+    runs-on: windows-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)